srcpkgs/docker-engine/template

```diff
@@ -0,0 +1,60 @@
+# Template file for 'docker-engine'
+pkgname=docker-engine
+version=20.10.5
+revision=1
+archs="aarch64* armv[67]l* x86_64* ppc64le*"
+create_wrksrc=yes
+build_style=go
+go_import_path="github.com/docker/docker"
+hostmakedepends="pkg-config tar"
+makedepends="libbtrfs-devel device-mapper-devel libseccomp-devel"
+depends="containerd runc iptables xz git"
+short_desc="Pack, ship and run any application as a lightweight container - engine"
```

Comment:
`short_desc="Engine for the Docker container runtime"` is parallel to my proposal for the `docker-cli` description and avoids marketing speak.